elasticsearch/class-elasticsearch.php

```diff
@@ -56,4 +61,22 @@ public function filter__ep_index_name( $index_name, $blog_id, $indexables ) {
 
 		return $index_name;
 	}
+
+	public function filter__ep_do_intercept_request( $request, $query, $args, $failures ) {
+		$fallback_error = new \WP_Error( 'vip-elasticsearch-upstream-request-failed', 'There was an error connecting to the upstream Elasticsearch server' );
+
+		$request = vip_safe_wp_remote_request( $query['url'], $fallback_error, 3, 1, 20, $args );
+	
+		return $request;
+	}
+
+	public function filter__jetpack_active_modules( $modules ) {
+		// Filter out 'search' from the active modules. We use array_filter() to get _all_ instances, as it could be present multiple times
+		$filtered = array_filter ( $modules, function( $module ) {
+			return 'search' === $module ? false : true;
```

Comment:
Minor: the ternary plus flipped return is a bit hard to read. I would simplify with an `if`:

```suggestion
			if ( 'search' === $module ) {
				return false;
			}
			return true;
```